src/ReactWrapperComponent.jsx

```diff
@@ -54,7 +54,7 @@ export default function createWrapperComponent(node, options = {}) {
       const component = this._reactInternalInstance._renderedComponent;
       const inst = component.getPublicInstance();
       if (inst === null) {
-        return component;
+        return component._instance;
```

Comment:
All of the methods that call `findDOMNode` are wrapped in `single` calls which pass in `this.node` which is returned by `getWrappedComponent` so this seems like the most logical place to make the change.

Unless there's another method to get the instance for SFCs specifically none of the methods dependant on `findDOMNode` will work (`html`, `text`, `simulate`)